TranslatorReasonerAPI.yaml

```diff
@@ -53,6 +54,32 @@ paths:
                     'biolink:Gene':
                       - biolink:directly_interacts_with
                       - biolink:decreases_activity_of
+  /knowledge_map:
+    get:
+      tags:
+        - knowledge_map
+      summary: Knowledge-map representation of this knowledge source.
+      responses:
+        '200':
+          description: >-
+            Returns knowledge-map representation of this knowledge source.
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/KnowledgeMap'
+  /info:
```

Comment:
I'm not convinced that we need this. I think that this information should be available in the service's OpenAPI schema itself.

Reply:
Agreed, /info was removed and relevant information will be provided by x-translator and x-trapi tags